index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;500;600;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Geospatial World Forum</title>
+  </head>
+  <body>
+    <!------- Navbar ---- -->
+    <nav>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+
+      <a href="index.html" class="logo">GWF 2023</a>
+      <ul class="nav__menu">
+        <li class="nav__item">
+          <a href="about.html" class="nav__link">About</a>
+        </li>
+        <li class="nav__item"><a href="#" class="nav__link">Program</a></li>
+        <li class="nav__item"><a href="#" class="nav__link">Join</a></li>
+        <li class="nav__item"><a href="#" class="nav__link">Sponsor</a></li>
+        <li class="nav__item"><a href="#" class="nav__link">News</a></li>
+      </ul>
+
+      <div class="nav__campaign">
+        <span>GWF Campaign</span>
+      </div>
+    </nav>
+
+    <!------ Headline section ----->
+    <section id="headline">
+      <h2>"Hello! Geospatial world!"</h2>
+      <h1>
+        GEOSPATIAL WORLD <br class="headline__br1" />GWF FORUM
+        <br class="headline__br2" />
+        2023
+      </h1>
+      <p class="headline__body">
+        Now in its 14th edition, Geospatial World Forum (GWF) has become a
+        not-to-be-missed conference for the professionals engaged in geospatial
+        and allied sectors. It is a collaborative and interactive platform that
+        demonstrates the collective and shared vision of the global geospatial
+        community.
+      </p>
+      <h3 class="headline__date">2023.05.02(TUE) ~ 05(FRI)</h3>
+      <span class="headline__location"> @ Rotterdam, The Netherlands </span>
+    </section>
+
+    <!------ Main section ----->
+    <section id="main">
+      <h3 class="main__header">Main Program</h3>
+      <span class="bar--secondary"></span>
+
+      <div class="main__events">
+        <div class="main__info">
+          <img src="images/icon1.svg" alt="icon" class="main__icon" />
+          <h3 class="main__heading">Lecture</h3>
+          <p class="main__details">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsam
+            voluptate recusandae nemo repellat animi odit?
+          </p>
+        </div>
```

Comment:
- [ ] Please kindly wrap individual event with an `article `tag according to the best practises. The `article `tag specifies independent, self-contained content and its semantic unlike `div`. 👍